app/api/__init__.py

```diff
@@ -43,11 +45,12 @@
           '/microlocations/<int:id>/relationships/event')
 
 # sessions
-api.route(SessionList, 'session_list', '/sessions', '/events/<int:id>/sessions')
+api.route(SessionList, 'session_list', '/sessions', '/events/<int:id>/sessions', '/tracks/<int:track_id>/sessions')
```

Comment:
For this, you have to create one more method, similar to this   
```python
def query(self, view_kwargs):
        query_ = self.session.query(Sponsor)
        if view_kwargs.get('event_id') is not None:
            query_ = query_.join(Event).filter(Event.id == view_kwargs['event_id'])
        return query_
```
This was for */events/<int:event_id>/sponsors*